mpileup.c

```diff
@@ -1288,12 +1302,30 @@ int main_mpileup(int argc, char *argv[])
         switch (c) {
         case 'x': mplp.flag &= ~MPLP_SMART_OVERLAPS; break;
         case  1 :
-            mplp.rflag_require = bam_str2flag(optarg);
-            if ( mplp.rflag_require<0 ) { fprintf(stderr,"Could not parse --rf %s\n", optarg); return 1; }
+            fprintf(stderr,"The option --rf has been renamed, please use --if instead\n");
+            mplp.rflag_incl = bam_str2flag(optarg);
+            if ( mplp.rflag_incl<0 ) { fprintf(stderr,"Could not parse --if %s\n", optarg); return 1; }
+            break;
+        case  17 :
```

Comment:
This could be simplified, with case 1 being an fprintf and then immediately falling into the case 17 as 1 and 17 are deliberately identical.